tests/framework/test_utilities.py

```diff
@@ -46,6 +46,16 @@ def test_rate_to_probability(rate: float, time_scaling_factor: float) -> None:
     expected == prob
 
 
+@pytest.mark.parametrize("rate", [0.75, pd.Series([0.75, 0.5, 0.25]), 0.1])
+def test_rate_to_probability_default_time_scaling(rate: float) -> None:
+    expected = rate
+    prob = rate_to_probability(rate)
+    if isinstance(expected, float):
```

Comment:
You won't need this branching pattern, because all calls will return an `np.array`.